### ISSUE TITLE
-Force and -NoClobber contradictory description

### ISSUE THREAD

**Issue by metablaster** [MicrosoftDocs/windows-powershell-docs#2364]
These 2 parameters are contradictory, NoClobber says Force will overwrite read-only file, Force says it won't do it.

`-Force` will overwrite an existing certificate file, **unless the Read-only or hidden attribute is set**
`-NoClobber` This parameter takes precedence over the Force parameter, **which permits this cmdlet to overwrite an existing certificate file, even if it has the Read-only attribute set**.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6c147fc5-e5c9-81ee-43b7-17638832572b
* Version Independent ID: aedf27ba-abfc-7e5d-4c69-cbf370fdac2b
* Content: [Export-Certificate (PKIClient)](https://docs.microsoft.com/en-us/powershell/module/pkiclient/export-certificate?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/pkiclient/Export-Certificate.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/pkiclient/Export-Certificate.md)
* Product: **w10**
* Technology: ****
* GitHub Login: @Kateyanne
* Microsoft Alias: **v-kaunu**

**Comment by velkovb** [comment]
@metablaster 
Thank you for your feedback. It seems like the the `-NoClobber` parameter description is actually the correct one. We will update the article with the correct information.

**Comment by metablaster** [comment]
Thank you for taking the time!
Should I close this issue or you handle it automatically?

**Comment by velkovb** [comment]
It will be closed automatically once the PR is merged.
